sql/migrations/20210704100709_world.sql

```diff
@@ -0,0 +1,61 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20210704100709');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20210704100709');
+-- Add your query below.
+
+-- Fix Dorius Stonetender pathing.
+DELETE FROM mangos.script_waypoint WHERE entry = '8284';
```

Comment:
Don't include the database name in the query, since other people might have it named differently. It should be just "script_waypoint" without the "mangos." part.

Reply:
Done, cheers